conference/src/main/java/com/kyriba/conference/management/service/HallServiceImpl.java

```diff
@@ -1,47 +1,70 @@
 package com.kyriba.conference.management.service;
 
 import com.kyriba.conference.management.api.dto.HallRequest;
+import com.kyriba.conference.management.api.dto.HallResponse;
```

Comment:
so 'service' depends on 'api' and vice versa. Correct?

Reply:
domain package should be independent, services package depends on domain, api package depends on root services package and dto subpackage in domain package